packages/design-system/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -24,6 +26,45 @@ export class HelpDrawer extends React.PureComponent {
 
   componentDidMount() {
     if (this.headingRef) this.headingRef.focus();
+
+    const eventAction = 'onComponentDidMount';
+
+    /* Send analytics event for helpdrawer open */
+    this.sendAnalyticsEvent(eventAction);
+  }
+
+  componentWillUnmount() {
+    const eventAction = 'onComponentWillUnmount';
+
+    /* Send analytics event for helpdrawer close */
+    this.sendAnalyticsEvent(eventAction);
+  }
+
+  sendAnalyticsEvent(eventAction) {
+    const analyticsOverrides = get(this.props.analytics, eventAction);
```

Comment:
Instead of exporting `sendTealiumEvent`, I wonder if we could export this entire `sendAnalyticsEvent` function as a helper. This way the boilerplate in components could be reduced even more. I.e.

```
sendAnalyticsEvent(overrides, defaultPayload) {
  const analyticsDisabled = overrides === false;
  
  if (window.utag && !analyticsDisabled) {
      const payload = merge(payload, analyticsOverrides);
      sendTealiumEvent(payload);
    }
}
```

And then usage would just be:
```
componentWillUnmount() {
    const eventAction = 'onComponentWillUnmount';

    /* Send analytics event for helpdrawer close */
    this.sendAnalyticsEvent(get(this.props.analytics, eventAction), get(DefaultAnalytics, eventAction));
  }
  ```